src/components/button/button.tsx

```diff
@@ -16,7 +16,7 @@ export class Button
   implements GxComponent, DisableableComponent, VisibilityComponent {
   constructor() {
     this.renderer = new ButtonRender(this, {
-      handleClick: this.handleClick.bind(this)
+      handleClick: !this.disabled ? this.handleClick.bind(this) : null
```

Comment:
What happens if `disabled` is changed in runtime?

Reply:
mmm... I thought it will be great but I didn't think in change the `disabled` in runtime. I'll remove that change

Reply:
Fixed in 09ae8dc